lib/cli/lib/helpers.js

```diff
@@ -197,15 +197,59 @@ export function installDeps(options) {
   done();
 }
 
-export async function installBabel(npmOptions, packageJson) {
+/**
+ * Add dependencies to a project using `yarn add` or `npm install`.
+ *
+ * @param {Object} npmOptions contains `useYarn` which we use to determine how we install packages.
+ * @param {Array} dependencies contains a list of packages to add. if you're trying to install dev dependencies you can append `'-D'` to the array.
+ * @example
+ * installDependencies(npmOptions, [
+ *   `@storybook/react@${storybookVersion}`,
+ *   `@storybook/addon-actions@${actionsVersion}`,
+ *   `@storybook/addon-links@${linksVersion}`,
+ *   `@storybook/addons@${addonsVersion}`,
+ *   '-D',
+ * ]);
+ */
+export function installDependencies(npmOptions, dependencies) {
```

Comment:
Is there maybe some npm package already doing this, so we don't have to maintain this code ourselves?

Reply:
I'll look into this in a bit, but my original thought was that we're currently relying on `cross-spawn` which looks like it handles Windows OS, so any other package we use would also need to meet that standard

Reply:
Either way, I propose moving the `-D` option out of the `dependencies` list and into `npmOptions`

Reply:
So looked at a few packages that might do this:
- [`npm-programmatic`](https://www.npmjs.com/package/npm-programmatic): seems heavier than we need. also not sure if it supports cross platform
- [`install-packages`](https://www.npmjs.com/package/install-packages): seems like exactly what we'd need, it also uses cross-spawn. less popular than the other option (weekly downloads/dependents)

My problem with both is that they're not highly active modules, and they seem like overkill for a simple function. So if it were me I'd stick to just `cross-spawn`, but I can go ahead and switch over to either of them if y'all would prefer 👍

Reply:
As for moving `-D` into `npmOptions`, that's a great idea, will do

Reply:
Looks like this got merged prematurely cc @ndelangen. Do you mind issuing that fix in a second PR?

Reply:
oh just saw this, PR incoming 👍